firefox-ios/firefox-ios-tests/Tests/XCUITests/HomepageSearchBarTests.swift

```diff
@@ -267,6 +267,23 @@ final class HomepageSearchBarTests: FeatureFlaggedTestBase {
         mozWaitForElementToExist(app.textFields[searchTextFieldA11y])
     }
 
+    func test_navigateBack_FromWebpageToHomepage_forTopToolbar_showsSearchBarExperimentOn() {
```

Comment:
We use camel case in XCUITest name. The underscore is for separate the `ExperimentOn` and `ExperimentOff` flag.

> Test Naming
> Suffix
> Adding the name of the experiment with its value at the end works to easily see the test purpose as well as its result. Cons is that the test name is very long. Example:
> testTopSitesRemoveAllExceptPinnedClearPrivateData_tabTrayExperimentOff()
> testTopSitesRemoveAllExceptPinnedClearPrivateData_tabTrayExperimentOn()

Reply:
Thanks @clarmso!

Where did you pull the documentation above from? We like to use `_` for unit tests, but we haven't been consistent on this. When @yoanarios, maybe we can chat on a consistent naming convention for before unit tests and UI tests or diverge if there's a compelling reason?

Reply:
See "Mobile Test Eng: Automated testing with Experiments" ➡️ Conclusion ➡️ Solution
https://docs.google.com/document/d/1Z3mbTUysScNcQQsqLPWf-gPpZG8-IqoCVo9ABmayXRU/edit?tab=t.y1x95srta7ir